src/entity/learning-object/submittable-learning-object.ts

```diff
@@ -122,12 +122,21 @@ export namespace SubmittableLearningObject {
     }
   }
   export function validateDescription(description: string) {
+    // Matches anything that does not have a closing HTML tag.
+    // Example: https://regex101.com/r/qDSGWv/1
+    const nonTerminatingTagRegex = /<([a-z]+)(\s[^>]*)?(?<!\/)>(?![\s\S]*<\/\1>)/gi;
+
     if (!description || !description.trim()) {
       throw new EntityError(
         SUBMITTABLE_LEARNING_OBJECT_ERRORS.INVALID_DESCRIPTION,
         'description',
       );
     }
+
+    // st-editor will always close HTML tags, so theoretically we shouldn't hit this.
```

Comment:
This is only because the st-editor was mentioned. Would we need to catch this for the notes field too?

Reply:
done